test/font_test.py

```diff
@@ -36,6 +36,9 @@ def setUp(self):
     def tearDown(self):
         pygame_font.quit()
 
+    def test_get_sdl_ttf_version(self):
+        pygame_font.get_sdl_ttf_version()
```

Comment:
Might as well add a test if this function actually returns a string or tuple with 3 ints.